packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -49,6 +49,7 @@ export type FacetSearchProps = {
 };
 
 export class FacetSearch {
+  private static ShowMoreResults = 1;
```

Comment:
Why is this property `static` and capitalized? I would also prefer to use a noun rather than a verb for the name to not suggest that it is a function.